kcmdline.c

```diff
@@ -0,0 +1,89 @@
+#include "kcmdline.h"
+
+static const CHAR8 allowed[6][12] = {
+	"^console=",
+	"^root=oci:",
+	"root=atomix",
+	"ro",
+	"quiet",
+	"verbose"
```

Comment:
Can you add crashkernel?  Is that safe?

Reply:
Unless you have a mechanism to verify an arbitrary crash-kernel it seems like it wouldn't be a safe option ... just my unsolicited two cents :)

Reply:
Well just allowing crashkernel=256M would probably be fine.  Or if the long road plan is to allow "merging" the custom args with the cmdline in the smoosh - but I don't think we know how to do that sanely.

Reply:
Oh and - hey Paul :) \o

(thanks for your input)